frontend/src/assets/styles/global.scss

```diff
@@ -13,13 +13,9 @@
 
 /* Vuetify overrides */
 
-.v-app-bar.v-app-bar--fixed {
-  z-index: 100 !important;
-}
-
-.v-btn {
-  overflow: hidden;
-}
+// .v-app-bar.v-app-bar--fixed {
+//   z-index: 100 !important;
+// }
```

Comment:
What's the reason for commenting instead of removing? There seems to be a bit of that in this file.

Reply:
@aweebs I originally commented the lines during the initial stages of the migration so I could come back to them quickly if needed, but at this point I agree it's better to start cleaning up the styles and use the git history if it's ever needed again.

Most are hacks to fix bugs in Vuetify 2, and all of them (at least I didn't found any that it isn't) are unnecessary now. I made a complete task for cleaning them up in the Vue 3 PR.

TLDR: Will remove them 👍🏻.